tools/rum-models-generator/run.sh

```diff
@@ -50,30 +68,49 @@ cd -
 swift build --configuration release
 GENERATOR=".build/x86_64-apple-macosx/release/rum-models-generator"
 
-# Generate file in temporary location
+# Generate RUM models (Swift) file in temporary location
+mkdir -p ".temp"
+GENERATED_SWIFT_FILE=".temp/RUMDataModels.swift"
+$GENERATOR generate-swift --path "rum-events-format/schemas" > $GENERATED_SWIFT_FILE
+echo "// Generated from https://github.com/DataDog/rum-events-format/tree/$SHA" >> $GENERATED_SWIFT_FILE
```

Comment:
Nitpick: I assume this reference is meant for us human, if that's the case maybe prepend that comment to the beginning of the file to make any manual checks faster.

Reply:
In `RUMModelsGenerator.swift`, when building the actual output for `*.swift` and `*+objc.swift` files, we print this comment along with the license header, `imports` etc.:
```swift
// This file was generated from JSON Schema. Do not modify it directly.
```
I think this is exactly what you mean, no?

Reply:
It's both for us and for `run.sh` to validate generated files. If running the tool with `verify` option (instead of `generate`), it reads the tail of the generated file, captures this `$SHA`, creates temporary models for that `$SHA` and compares if the `diff <original> <temporary>` is empty.

It is just easier to print it at the end of the file than in the middle. Printing in the middle would require understanding the file structure or at least printing-in some token to replace it later later (e.g. `s/<SHA-HERE>/$SHA/g`) here in `.sh`.

At the very top of each file we print this:
```swift
// This file was generated from JSON Schema. Do not modify it directly.
```
to highlight that this file is auto-generated. To find the signature, we need to scroll to the bottom.

I can generate the signature at the top of the file (after the license), but this would require some work - I'm not sure if it's worth it 🙂. WDYT?